### DISCUSSION TITLE
On GitHub for Android latest version, Myanmar Unicode Rendering went wrong.

### DISCUSSION THREAD

**Discussion by aswedaw** [community/community#77845] [Mobile]
Would you please use former fallback font and follow the customized font of our devices as before. MIUI 12.5.3 China ROM devices, use NotoSans Myanmar Zawdecode as the default font of the devices. It's
non-Unicode.
ကျောက်မဲ၊ အိပ်မက်၊ လက်မည်းကြီး
For example, these words went wrong when we tap SEND key.

**Comment by dakshithadissanayaka** [comment]
Sure, here is a summary of the discussion and a possible solution:

**Issue:**

A user is suggesting that GitHub implement private messaging for a more convenient way to communicate within the platform.

**Possible Solution:**

Implementing private messaging could enhance communication and collaboration among GitHub users. Here are some potential benefits of integrating private messaging into GitHub:

1. **Direct and Targeted Communication:** Private messaging would enable direct and targeted communication between users, facilitating discussions on specific projects, code reviews, or collaboration tasks without cluttering public repositories.

2. **Real-time Collaboration:** Private messaging could foster real-time collaboration, allowing users to exchange ideas, share feedback, and resolve issues promptly within the GitHub platform.

3. **Enhanced Community Engagement:** Private messaging could strengthen community engagement by enabling users to connect with like-minded individuals, seek mentorship, and build relationships within the GitHub ecosystem.

4. **Privacy and Security:** Private messaging could provide a secure and private channel for sensitive discussions, confidential project information, and personal interactions.

Here are some possible implementation considerations for private messaging on GitHub:

1. **Integration with Existing Features:** Private messaging could be integrated into the existing GitHub workflow, allowing users to seamlessly transition from public discussions to private conversations.

2. **User Controls and Permissions:** Granular user controls and permissions would be essential to manage privacy settings, block unwanted messages, and maintain control over communication preferences.

3. **Contextual Awareness:** Private messaging should be contextually aware, allowing users to initiate conversations based on specific repositories, issues, or pull requests.

4. **Security and Privacy:** Robust security measures should be implemented to protect user data and ensure the privacy of private conversations.

5. **Notification Preferences:** Users should have the option to customize notification settings for private messages, ensuring they receive relevant updates without being overwhelmed.

Implementing private messaging on GitHub could provide a valuable addition to the platform, enhancing communication, collaboration, and community engagement among users.

**Reply by Akash1134** [reply]
Hi @dakshithadissanayaka ,

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by iskae** [comment]
Hi @aswedaw 👋 

Thank you so much for reaching out ❤️ 

We have investigated this and unfortunately there is not much that we can do from our side 😞 For our native views we are following the device default fonts, but we also have WebViews in our apps that we have no control over. Can you please try updating your default WebView on your device?

Cheers!

**Reply by aswedaw** [reply]
After updating Android System WebView version 119, it went wrong.
